lib/vrl/tests/tests/diagnostics/invalid_if_condition_type.vrl

```diff
@@ -9,9 +9,10 @@
 #   │    this predicate must resolve to a boolean
 #   │    instead it resolves to "string"
 #   │
-#   = hint: coerce the value using one of the coercion functions
-#   = learn more at: https://errors.vrl.dev/102
-#   = see language documentation at: https://vector.dev/docs/reference/vrl/
+#   = hint: coerce the value to the required type using a coercion function
+#   = see if expression documentation at https://vrl.dev/expressions/#if
```

Comment:
It would be slightly more readable on the first read if that `if` was in some quotations.

Reply:
I agree, and I actually think it should read like this: "see documentation for ___ at ___." Good catch.